fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/SettingsSubMenuAutofillRobot.kt

```diff
@@ -314,14 +410,22 @@ private val toolbarDeleteAddressButton = itemWithResId("$packageName:id/delete_a
 private val cancelDeleteAddressButton = onView(withId(android.R.id.button2)).inRoot(RootMatchers.isDialog())
 private val confirmDeleteAddressButton = onView(withId(android.R.id.button1)).inRoot(RootMatchers.isDialog())
 
+private val creditCardsSectionTitle = itemContainingText(getStringResource(R.string.preferences_credit_cards))
+private val saveAndAutofillCardsOption = itemContainingText(getStringResource(R.string.preferences_credit_cards_save_and_autofill_cards))
+private val saveAndAutofillCardsSummary = itemContainingText(getStringResource(R.string.preferences_credit_cards_save_and_autofill_cards_summary))
+private val syncCardsAcrossDevicesButton = itemContainingText(getStringResource(R.string.preferences_credit_cards_sync_cards_across_devices))
 private val addCreditCardButton = mDevice.findObject(UiSelector().textContains(getStringResource(R.string.preferences_credit_cards_add_credit_card)))
+private val savedCardsToolbarTitle = itemContainingText(getStringResource(R.string.credit_cards_saved_cards))
+private val editCardToolbarTitle = itemContainingText(getStringResource(R.string.credit_cards_edit_card))
 private val manageSavedCardsButton = mDevice.findObject(UiSelector().textContains(getStringResource(R.string.preferences_credit_cards_manage_saved_cards)))
 private val cardNumberTextInput = mDevice.findObject(UiSelector().resourceId("$packageName:id/card_number_input"))
 private val nameOnCardTextInput = mDevice.findObject(UiSelector().resourceId("$packageName:id/name_on_card_input"))
 private val expiryMonthDropDown = mDevice.findObject(UiSelector().resourceId("$packageName:id/expiry_month_drop_down"))
 private val expiryYearDropDown = mDevice.findObject(UiSelector().resourceId("$packageName:id/expiry_year_drop_down"))
 private val savedCreditCardNumber = mDevice.findObject(UiSelector().resourceId("$packageName:id/credit_card_logo"))
-private val deleteCreditCardButton = mDevice.findObject(UiSelector().resourceId("$packageName:id/delete_credit_card_button"))
+private val toolbarDeleteCreditCardButton = mDevice.findObject(UiSelector().resourceId("$packageName:id/delete_credit_card_button"))
+private val toolbarSaveCreditCardButton = itemWithResId("$packageName:id/save_credit_card_button")
+private val deleteCardButton = itemContainingText(getStringResource(R.string.credit_cards_delete_card_button))
```

Comment:
Let's just be consistent with having `credit` in our function names.
`deleteCreditCardButton`

Reply:
Done !